CONTRIBUTING.md

```diff
@@ -27,6 +47,67 @@ pull request reviewed, the reviewer(s) may ask you to complete additional
 work, tests, or other changes before your pull request can be ultimately
 accepted.
 
+# For maintainers
+
+## Setting up environment
+
+Prerequisites:
+
+- Dart SDK installed in the path, [standalone][dart_standalone_link], or included in the [Flutter SDK][dart_on_flutter_link].
+- [Mason CLI][mason_install_link] (to run and test the `bricks`);
+- [Node.js][node_js_dowload_link] (For the VS Code extension and the docs site);
```

Comment:
I prefer forwarding the users to read the [VS Code's contributing file](https://github.com/VeryGoodOpenSource/dart_frog/blob/68b290698068e5a2446f1cea7fb68b8921861f07/extensions/vscode/CONTRIBUTING.md#setting-up-your-local-development-environment). Where it is documented what version of Node.js will they need and any other dependencies.

I'm not sure if the same Node.js version is compatible with the docs site to.

Reply:
Sure, how should we word that?

Reply:
```suggestion
- [Node.js][node_js_dowload_link], for working with the VS Code extension or the documentation website. Please refer to their CONTRIBUTING files for further installation requirements.
- [Yarn](https://classic.yarnpkg.com/lang/en/docs/install/), for working with the documentation website.
```